<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cezar</class>
 <widget class="QDialog" name="cezar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Шифр Цезаря</string>
  </property>
  <widget class="QTextEdit" name="cezartext">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Пропишите полный путь до файла&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="cezarLineT1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="cezarLineKey">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="cezarLineT2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="cezarlabel1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Исходный файл</string>
   </property>
  </widget>
  <widget class="QLabel" name="cezarlabel2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ключ</string>
   </property>
  </widget>
  <widget class="QLabel" name="cezarlabel3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Конечный файл</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cezarButton1">
   <property name="geometry">
    <rect>
     <x>247</x>
     <y>120</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Шифровать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cezarButton2">
   <property name="geometry">
    <rect>
     <x>247</x>
     <y>170</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Дешифровать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="helpButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>30</y>
     <width>21</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
